server/api/api/endpoints/functions.py

```diff
@@ -1086,16 +1070,14 @@ def _is_nuclio_deploy_status_changed(
 
 def create_model_monitoring_stream(
     project: str,
-    function,
     stream_path: str,
     monitoring_application: bool = None,
     access_key: str = None,
+    stream_args: dict = None,
```

Comment:
`stream_args` is not an actual dict
```suggestion
    stream_args: mlrun.mlconf.Config = None,
```